scene/gui/code_edit.cpp

```diff
@@ -1602,6 +1617,27 @@ bool CodeEdit::can_fold_line(int p_line) const {
 		return false;
 	}
 
+	// Check for code region.
+	if (is_line_code_region_end(p_line)) {
+		return false;
+	}
+	if (is_line_code_region_start(p_line)) {
+		int region_level = 0;
+		/* Check if there is a valid end region tag */
```

Comment:
```suggestion
		// Check if there is a valid end region tag.
```
[Comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide) nitpick